content/attacks/posts/2024-02-23-BitForex.md

```diff
@@ -0,0 +1,58 @@
+---
+date: 2024-02-23
+target-entities: BitForex
+entity-types:
+  - Exchange
+  - Custodian
+attack-types:
+  - Exit Scam
+  - Wallet Hack
+title: "BitForex's Suspicious Outflows: Hack or Exit Scam?"
+loss: 56500000
+---
+
+## Summary
+
+Since February 23, 2024, BitForex, a crypto exchange operational since 2017, ceased processing withdrawals amidst unexplained outflows of about $56.5M worth of crypto from its hot wallets. The absence of communication from BitForex, coupled with the recent departure of its CEO Jason Luo, has raised concerns over a potential inside job or exit scam. 
+
+## Attackers
+
+The identity of the attacker is unknown. The following addresses are associated with this attack:
+
+- Ethereum Wallet:
+  - [0xdcacd7eb6692b816b6957f8898c1c4b63d1fc01f](https://etherscan.io/address/0xdcacd7eb6692b816b6957f8898c1c4b63d1fc01f)
+    
+- Tron Wallet:
+  - [TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o](https://tronscan.org/#/address/TQcnqaU4NDTR86eA4FZneeKfJMiQi7i76o)
+   
+- Bitcoin Wallet:
+  - [3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2](https://www.blockchain.com/explorer/addresses/btc/3DbbF7yxCR7ni94ANrRkfV12rJoxrmo1o2)
+
+## Losses
+
+The total loss from the BitForex hack amounted to approximately [$56.5 million](https://twitter.com/zachxbt/status/1762028433574650347), distributed across the following networks:
+
+- Ethereum:
+  -  471,414 TRB (54,200,000 USD)
+  -  148 ETH (444,000 USD)
+  -  258,700 USDC
+  -  40,771 USDT
+
+- Tron
+  - 657,698 USDT
+  - 44,000 TRX (6072 USD)
+
+- Bitcoin
+  - 5.7 BTC (290,000 USD)
+
+## Timeline
+
+- **January 31, 2024, 02:07 PM UTC:** Exchange CEO Jason Luo [steps down](https://support.bitforex.com/hc/en-us/articles/28260960127385-Jason-Stepped-Down-as-CEO-of-BitForex).
+- **February 21, 2024, 08:16:23 AM UTC:** The [first malicious](https://etherscan.io/tx/0xee8a0e425670b53e5066451362324fb6ef36fb5507a525567fc8c9b68a03709f) transaction occurred.
+- **February 23, 2024, 07:12:59 PM UTC:** The withdrawal of funds from the exchange [began](https://etherscan.io/tx/0xb1ef6176112c8b019192b5e627732afee8930810801961ded081720e74544753) and the exchange stopped operating without warning.
```

Comment:
Unnecessary line. You can add a source that posted about website shutdown instead.